SimPEG/EM/Base.py

```diff
@@ -129,83 +172,83 @@ def MeMuI(self):
             Inverse of :code:`MeMu`
         """
         if getattr(self, '_MeMuI', None) is None:
-            self._MeMuI = self.mesh.getEdgeInnerProduct(self.curModel.mu, invMat=True)
+            self._MeMuI = self.mesh.getEdgeInnerProduct(self.mu, invMat=True)
         return self._MeMuI
 
-
     # ----- Electrical Conductivity ----- #
-    #TODO: hardcoded to sigma as the model
+    # TODO: hardcoded to sigma as the model
```

Comment:
Well, I suppose, sigma is still the only possible model, but it is a bit more explicit now!